etc/spgw_c.json

```diff
@@ -51,7 +51,8 @@
      "default_dns_sec_ipv4_address" : "@DEFAULT_DNS_SEC_IPV4_ADDRESS@",
      "default_dns_ipv6_address"     : "2001:4860:4860::8888",
      "default_dns_sec_ipv6_address" : "2001:4860:4860::8844",
-     "force_push_protocol_configuration_options" : @PUSH_PROTOCOL_OPTION@
+     "force_push_protocol_configuration_options" : @PUSH_PROTOCOL_OPTION@,
+     "ue_mtu_ipv4": @UE_MTU_IPV4@
```

Comment:
@lionelgo 

Is there a default value we can put in the entrypoint?

`1500`  or `1464` as you have in the code?

Raphael

Reply:
Yes exactly, for people who know what they are doing regarding the mtu on s1u link

Reply:
OK I push a workaround in entrypoint